apps/web/src/components/Footer/index.tsx

```diff
@@ -112,26 +112,69 @@ const Footer: FC<FooterProps> = ({ links, socials, ctaCopy, ctaLabel, message })
             {ctaLabel}
           </Link>
         </div>
-        <p className="landscape:self-end">{message}</p>
+        <p className="landscape:self-end">
+          <Link
+            href="https://www.mozilla.org/about/legal/terms/firefox/?utm_source=billionaireblastoff.firefox.com&utm_medium=referral&utm_campaign=footer"
+            aria-label="Go to Privacy Policy page"
+            title="Go to Privacy Policy page"
+            className="underline"
+            target="_blank"
+          >
+            Privacy Policy
+          </Link>{' '}
+          &bull;{' '}
+          <Link
+            href="https://www.mozilla.org/en-US/privacy/websites/?utm_source=billionaireblastoff.firefox.com&utm_medium=referral&utm_campaign=footer"
+            aria-label="Go to Terms and Conditions page"
+            title="Go to Terms and Conditions page"
+            className="underline"
+            target="_blank"
+          >
+            Terms & Conditions
+          </Link>
+        </p>
       </div>
 
-      <div className="flex flex-row justify-between font-medium text-xs10 landscape:text-base mt-2 pt-2 border-t-2 border-common-ash landscape:mt-6 landscape:pt-6">
+      <div className="flex flex-col landscape:flex-row gap-1 justify-between font-medium text-xs10 landscape:text-base mt-2 pt-2 border-t-2 border-common-ash landscape:mt-6 landscape:pt-6">
         <span>@{currentYear.getFullYear()} Mozilla. All rights reserved.</span>
-        <span>
-          Built by{' '}
+        <span className="flex gap-x-2 items-center">
+          <span className="whitespace-nowrap">Powered by</span>
+          <Link
+            href="https://www.firefox.com/"
+            target="_blank"
+            title="Go to Firefox website"
+            className="inline-flex items-center -mb-1 pr-1 gap-2 font-bold"
+          >
+            <Image
+              src="/assets/images/logo-firefox.webp"
+              width={26}
+              height={26}
```

Comment:
@jhonymoncada when setting images using `width={}` and `height={}` you must also set CSS classes for width and height using `rem`s such as below, otherwise the scaling will be off.

```
className="w-[1.625rem] h-[1.625rem]"
```